config/locales/en/steps.yml

```diff
@@ -380,7 +380,7 @@ en:
           success_flash: You removed the investment
       investments_summary:
         edit:
-          page_title: investments summary list
+          page_title: Your client’s investments
```

Comment:
Oop, sorry @timpeat, I was wrong it actually is an even more annoying title as content wants it to match the heading. Is that possible to do?

https://docs.google.com/spreadsheets/d/1gdDpuyqWO6MJgUT1SOy0nbHd8AAdp1YebBzh1WDG040/edit#gid=0&range=D185

Same for savings

Reply:
Hmm, seems not. Maybe this is a good compromise